.github/workflows/build-hpc.yml

```diff
@@ -0,0 +1,104 @@
+name: build-hpc
+
+# Controls when the action will run
+on:
+
+  # Trigger the workflow on all pull requests
```

Comment:
Don't we want to run this on `main` after merge-pushes like we usually do?

Reply:
Thanks, good point! Fixed now